webrecorder/browsertrix
# Archive and crawl navigation improvements

- Visual alignment improvements to global navigation and archive tabs (i.e. crawls, crawl templates, members)
- Fix spinner/loading indicator alignment on archive view
- Separate crawl detail page into navigable sections in preparation for https://github.com/webrecorder/browsertrix-cloud/issues/134 and https://github.com/webrecorder/browsertrix-cloud/issues/142

A UX nice-to-have sometime down the road is having the home page be the user's default archive.

### Manual testing
1. Run app and go to an archive. Verify tabs still work as expected
2. Click into a crawl. Verify navigation on side works as expected

### Screenshots

**Overview (default):**
<img width="1033" alt="Screen Shot 2022-02-22 at 11 45 59 AM" src="https://user-images.githubusercontent.com/4672952/155207608-88a2361c-d67a-4db9-ae37-a1df859a4a8d.png">

**Recording:**
<img width="1018" alt="Screen Shot 2022-02-22 at 11 46 03 AM" src="https://user-images.githubusercontent.com/4672952/155207648-61e56c97-17b2-4ab0-b6f6-e9fc458392ae.png">

**Download:**
<img width="1020" alt="Screen Shot 2022-02-22 at 11 46 08 AM" src="https://user-images.githubusercontent.com/4672952/155207683-5a52551a-85ea-4657-acf3-4b1b7a32132d.png">

### Opinions
I called the watch/replay tab "Recording" to try and find a short tab name without adding a conditional "watch" or "replay". Another option I thought of is "View capture", unless @ikreymer you think "Watch or Replay" is the best?
